test/contracts/factory/evm_script_factory.js

```diff
@@ -113,6 +113,7 @@ contract('EVM Script Factory', ([permissionsRoot]) => {
 
       // The executor always uses 0x for the input and callscripts always have 0x returns
       const expectedExecutor = await evmScriptReg.getScriptExecutor(createExecutorId(1))
+      // Note
```

Comment:
This is irrelevant for Disptuables, but I saw a number of places where we had `input: null, returnData: null` when checking `ScriptResults`.

Should we be able to assert for these?

Reply:
This is because we are using `AppStubScriptRunner` and there is no input and return data, it is actually logging null values